### DISCUSSION TITLE
Search bar

### DISCUSSION THREAD

**Discussion by WaVePT** [MissingCore/Music#112] [Ideas]
Hello.
I would suggest for a new feature a search bar, to make searching for songs easier, that would make the user able to search the artists, songs, albums and playlists.
Best regards.

**Comment by cyanChill** [comment]
Search is coming in the next major update, if you're using the APK version, you can try it out early with the [`v2.0.0-rc.1`](https://github.com/MissingCore/Music/releases/tag/v2.0.0-rc.1) release.
